package.json

```diff
@@ -4,6 +4,7 @@
     "build": "npm run build --workspaces",
     "build-lib": "npm run build                 -w packages/ui-library -w packages/ui-library-react -w packages/ui-library-vue",
     "publish-lib": "npm publish --access public -w packages/ui-library -w packages/ui-library-react -w packages/ui-library-vue",
+    "publish-lib:dry": "npm publish --dry-run --access public -w packages/ui-library -w packages/ui-library-react -w packages/ui-library-vue",
```

Comment:
I suggest we align the `-w ...` parameters either in all scripts or in none.

Reply:
14e277d929c85e5914271881b1c5ff13de8f239a